Tutorials/Members-Registration-And-Logins/index.md

```diff
@@ -0,0 +1,245 @@
+
+![New Content Template](images/v8-06-RTE-Macro.png)
+
+Choose the Macro you created. Depending on how you would like to structure your signup/login pages, you can include one of the macros or place both of them next to each other.
+
+![Settings Menu](images/v8-07-RTE-Macro-Insert.png)
+
+Save and publish the new page. Now the Register/Login functionality should be rendered on the new page you made.
+
+![Create Content Template](images/v8-08-Login-Page-Frontend.png)
+
+With the above macros in place, you can now use the page to register new Members.
+Every registered person will show up in the Members section in the backoffice.
+
+![Select Content Type](images/v8-09-Backoffice-Members.png)
+
+Now, we have our signup and login functionality - though there is one partial view we have not used yet, the Login Status.
+We created it as a Partial View, not as a Macro, for a singular reason - we are going to include it directly in a template, not in a property editor in the Content section.
+This partial will show the current status - whether the person is logged in or not.
+A good place to put it would be either the Master page, or a navigation component.
+
+![Content Template Name Field](images/v8-10-Login-Partial-In-Nav.png)
+
+With this partial in place, provided we put it in navigation or a layout page, we will be able to see the login status no matter on which page we currently are. It also allows the member to log out.
+Do keep in mind that you might need to style the partial a bit to make it fit with the rest of the site.
+
+![New Content Template](images/v8-11-Login-Partial-Frontend.png)
+
+## Member-only pages/Restricted access
+
+Now that we have the options to:
+-register a member
+-log in a member
+-check current login status
+-log out a member
+
+We can also go a bit further and specify which parts of our website should be accessible to logged in members.
+To do this, head on over to the Member section in the Backoffice, and create a new Member Group.
+
+![Edit Content Template](images/v8-12-New-Member-Group.png)
+
+Give the group a name, and save it. Next, move over to the Members folder, click on any of the Members you  have, and assign the newly created Member Group to them.
+
+![Create From Template](images/v8-13-Member-Group-Assign.png)
+
+Almost there. Save the member, move to the Content section and find the page you would like to restrict. Right-click on that content node, and find the option "Restrict Public Access".
+
+![Select Template](images/v8-14-Restrict-Public-Access.png)
+
+You will be able to restrict access to a specific member, or a specific group. Choose the latter option.
+On this menu, you will be able to select the group that will have access to the page, the page with the login form, and the page that would be displayed if selected content would be inaccessible to the chosen group.
+
+![Select Template](images/v8-15-Restrict-Public-Access-Details.png)
+
```

Comment:
I think it would be worth mentioning here that the login and error pages are both mandatory in order to set this up.
I would suggest that you mention this before you start restricting access.
So, mention that in order to set up the public access you need to have a couple of pages ready: One for logging in and one to display for those who try to view a page they aren't allow to.

What do you think about that idea?

Reply:
I was structuring it as a step-by-step tutorial, so I thought by this point the reader would already have those pages 🤔 
Should I make it more obvious?

Reply:
Added a note.